examples/reference/elements/bokeh/Tiles.ipynb

```diff
@@ -79,7 +86,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "hv.Layout([ts().relabel(name) for name, ts in hv.element.tiles.tile_sources.items()]).opts(\n",
+    "valid_src = {k: v for k, v in hv.element.tiles.tile_sources.items() if v.__name__ != 'deprecated_tilesource_warning'}\n",
```

Comment:
Maybe we should have a variable in `hv.element.tiles` of the tile sources that aren't being deprecated?